coverage/cmdline.py

```diff
@@ -658,6 +677,12 @@ def command_line(self, argv):
                 show_contexts=options.show_contexts,
                 **report_args
             )
+        elif options.action == "lcov":
+            outfile = options.outfile
```

Comment:
I thought this pre-assignment was odd, but I see now that the other reporting clauses do it like that.  I'm changing them :)